android/app/src/main/java/org/fossasia/openevent/activities/MainActivity.java

```diff
@@ -274,51 +279,73 @@ private void doMenuAction(int menuItemId) {
         FragmentManager fragmentManager = getSupportFragmentManager();
         switch (menuItemId) {
             case R.id.nav_tracks:
+                addShadowToAppBar(true);
```

Comment:
@aakashkhatter1296 I believe this should be done after the transaction is complete as if the transaction is not complete, then the shadow won't be there